lib/bundles/inspec-compliance/api.rb

```diff
@@ -8,24 +8,30 @@
 module Compliance
   # API Implementation does not hold any state by itself,
   # everything will be stored in local Configuration store
-  class API
+  class API # rubocop:disable Metrics/ClassLength
     # return all compliance profiles available for the user
     def self.profiles(config)
-      url = "#{config['server']}/user/compliance"
+      config['automate'][0] ? url = "#{config['server']}/#{config['user']}" : url = "#{config['server']}/user/compliance"
```

Comment:
I am not fan of `config['automate'][0]`. We should ask directly for the information we need: `config['automate']['ent']`. For this case I propose config['server_type']